crates/node/ethstats/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "reth-node-ethstats"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[dependencies]
+
```

Comment:
one thing we definitely need is websocket support, reth-storage-api, tokio, tokio-tungstenite